pipeline.ipynb

```diff
@@ -64,40 +73,56 @@
    },
    "outputs": [],
    "source": [
+    "#Set up Initial Basic Parameters#\n",
     "minian_path = \".\"\n",
-    "dpath = \"./demo_movies\"\n",
+    "dpath = \"./demo_movies/\"\n",
```

Comment:
is this intended?

Reply:
No, this was by accident and was fixed in commit `a6727ba`